zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,17 @@
+Get Zulip notifications for your Honeybadger projects!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your honeybadger project's **settings** and select **Alert & Integrations menu**.
```

Comment:
Is it actually called "Alert & Integrations menu"?

Reply:
Yes, you can use the app to confirm. Also, you could refer to the [doc](https://docs.honeybadger.io/guides/integrations.html#rate-escalations) in which the video states the same.

Reply:
Resolved